metadata_schema.yaml

```diff
@@ -0,0 +1,558 @@
+# Description : Metadata used to describe a Terraform Module and its contents.
```

Comment:
Looks like observability metadata file is being copied here ... please update the file with atracker module info

Reply:
https://github.com/terraform-ibm-modules/terraform-ibm-atracker/pull/2/commits/b3f3fec5c337c68d0dedde3f8b9270f8d84ab493